index.html

```diff
@@ -0,0 +1,237 @@
+                <a class="nav-link" href="#">
+                  My Page
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">
+                  Logout
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </div>
+  </div>
+      
+  <!-- Navbar -->
+ <nav class="navbar navbar-expand-lg navbar-light bg-white container-fluid">
+  <button class="navbar-toggler mr-auto" type="button" data-toggle="collapse" data-target="#navbarNav"
+    aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <a class="navbar-brand logo" href="index.html"><img class="logo" src="./images/future (2) (1).png"/></a>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+        <li class="nav-item active">
+            <a class="nav-link about active" href="about.html">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link program" href="about.html">Program</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link join" href="about.html">Join</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link sponsor" href="about.html">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link news" href="about.html">News</a>
+          </li>
+    </ul>
+  </div>
+  <span class="campagin">CC Campagin</span>
+ 
+</nav>
+
+
+<section class="auto-layout">
+  <div class="container hero-container">
+    <div class="row">
+      <div class="col-12 col-md-6 hero-logo">
+        <img src="./images/theme-logo-mobile.png" alt="Mobile Logo" class="mobile-logo d-md-none">
+        <img src="./images/home-logo.png" alt="Desktop Logo" class="desktop-logo d-none d-md-block">
+      </div>
+      <div class="col-12 col-md-6 sub-team">
+        <p>Our event promotion services specialize in creating a captivating buzz, utilizing engaging social media campaigns and eye-catching advertisements to attract enthusiastic attendees and make your event unforgettable.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12 block">
+        <h3 class="event-date">2023.10.10(THU) ~ 12(SAT)</h3>
+        <small class="event-location">@ The Louvre Museum, Sydney Opera House, and more</small>
+      </div>
+    </div>
+    
+  </div>
+</section>
+
+
+<section class="custom-section">
+  <div class="container">
+    <div class="row align-items-center main-program-row">
+      <div class="col-12 text-center">
+        <h2 class="main-program-title">Main Program<span class="underline"></span></h2>
+      </div>
+    </div>
+    <div class="row card-row">
+      <div class="col-lg-1"></div>
+      <div class="col-lg-2 col-md-4 col-sm-12">
+        <div class="card">
+          <div class="card-content">
```

Comment:
- [ ] Hi, I noticed that you are using a div tag to wrap your card-content i.e Main program items. While this is not wrong, it is better to use a semantic HTML element like an article tag. This will make your code more readable and accessible, and also improve your SEO. An article tag represents a self-contained piece of content that can be independently distributed or reused. You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article. I suggest you replace your div tag with an article tag and see if it makes any difference. Let me know if you have any questions! 
- [ ] Please make above changes for all card contents.